src/Libraries/CoreNodeModels/DropDown.cs

```diff
@@ -78,6 +78,29 @@ public int SelectedIndex
             }
         }
 
+        public string getNameOfSelectedDropDownItem ()
```

Comment:
Methods in C# language is always of Pascal case, so I would recommend this to be named `GetSelectedItemText`.